plugin.video.alfa/core/httptools.py

```diff
@@ -550,8 +550,8 @@ def proxy_post_processing(url, proxy_data, response, **opt):
     data = ""
     if response["data"]:
         data = response["data"][:500]
-        if isinstance(data, bytes):
-            data = data.decode()
+        if PY3 and isinstance(data, bytes):
+            data = "".join(chr(x) for x in bytes(data))

```

Comment:
Esto solo es más confuso e innecesario. `data.decode()` hace lo mismo pero más eficiente, porque no convierte doblemente, es decir, no hace `bytes(bytes())` dado que el `if` ya dictamina que se está tratando con bytes.

Si lo que se quiere es que decodifique ignorando los errores, pues se hace `data.decode(errors='ignore')`. Y en cualquier caso, forzar la decodificación con este hack devolverá caracteres erróneos por la forma en que `chr` convierte números a caracteres asumiendo que son ordinales UTF-8. Véase [este hilo de stackoverflow](https://stackoverflow.com/questions/43564800/difference-between-chr-and-bytes-decode).

Haré la modificación, espero no tener que volverla a hacer. No quiero meterme en detalles porque la razón de que se deba de hacer esta verificación en primer lugar o está en la web que no devuelve la codificación correcta o está en el código más arriba que asigna ese `response["data"]`, y si lo encuentro más arriba terminaré por refactorizar más código.